modules/openthread/platform/radio_nrf5.c

```diff
@@ -0,0 +1,1810 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @file
+ *   This file implements the OpenThread platform abstraction
+ *   for radio communication utilizing nRF IEEE802.15.4 radio driver.
+ *
+ */
+
+#include <openthread/error.h>
+#define LOG_MODULE_NAME net_otPlat_radio
```

Comment:
We could have something different here, specific for nRF5 radio.

Reply:
Right, will change it :)